sdk/storage/azure-storage-file/HISTORY.md

```diff
@@ -1,6 +1,24 @@
 # Change Log azure-storage-file
 
 
+## Version 12.0.0b3:
+
+**New features**
+- Added FileSmbProperties REST Parity.
+- Added support for PutRangeThroughURL.
+
```

Comment:
- Added upload_range_from_url API to writes the bytes from one Azure File endpoint into the specified range of another Azure File endpoint.
- Added set_http_headers for directory_client, create_permission_for_share and get_permission_for_share APIs
- Added optional parameters smb properties related parameters for create_file*, create_directory* related APIs and set_http_headers API
- Updated get_properties for directory and file so that the response has SMB properties